### ISSUE TITLE
what is the difference between novel categories and base category?

### ISSUE THREAD

**Issue by seyeeet** [kjunelee/MetaOptNet#48]

**Comment by kjunelee** [comment]
Base categories usually refer to the categories used to train the meta-learner (a.k.a. meta training set). Novel categories are also referred to as meta test set.
